docs/source/concepts/cl_columns.rst

```diff
@@ -124,3 +124,93 @@ Contains a boolean value, with ``True`` indicating that the sample is likely to
 -----------
 Contains a score bounded between 0 and 1, which is used to determine whether a sample is an outlier. The lower the score of a sample, the more likely it is to be an outlier.
 
+
+Project Information Columns
+============
+
+
+You can also retrieve information/metadata related to the Cleanlab project itself.
```

Comment:
I think this line would be better if it were more descriptive. Maybe something like this?

For each project that you run in Cleanlab Studio, Cleanlab stores a number of metadata columns that can be retrieved using the Python API. Currently we store the following project information columns: